Aquality.Selenium.Template/Aquality.Selenium.Template/Forms/Pages/MainPage.cs

```diff
@@ -4,7 +4,7 @@ namespace Aquality.Selenium.Template.Forms.Pages
 {
     public class MainPage : BaseAppForm
     {
-        public MainPage() : base(By.XPath("//section[@class='testing-services-block']"), "Main page")
+        public MainPage() : base(By.XPath("//section[@class='heroMain heroMain--video-paused']"), "Main page")
```

Comment:
please don't refer to @class attribute in xpath locators like this; use contains